nip47/notifications/nip47_notifier_test.go

```diff
@@ -112,22 +120,32 @@ func TestSendNotification_PaymentReceived(t *testing.T) {
 	svc, err := tests.CreateTestService()
 	require.NoError(t, err)
 
-	app, ss, err := tests.CreateApp(svc)
+	app, cipher, err := tests.CreateAppWithPrivateKey(svc, "", "1.0")
 	assert.NoError(t, err)
-	doTestSendNotificationPaymentReceived(t, svc, app, ss)
+	doTestSendNotificationPaymentReceived(t, svc, app, cipher)
 }
 
-func TestSendNotification_Legacy_PaymentReceived(t *testing.T) {
+func TestSendNotification_LegacyVersion_PaymentReceived(t *testing.T) {
 	defer tests.RemoveTestService()
 	svc, err := tests.CreateTestService()
 	require.NoError(t, err)
 
-	app, ss, err := tests.CreateLegacyApp(svc, nostr.GeneratePrivateKey())
+	app, cipher, err := tests.CreateAppWithPrivateKey(svc, "", "0.0")
 	assert.NoError(t, err)
-	doTestSendNotificationPaymentReceived(t, svc, app, ss)
+	doTestSendNotificationPaymentReceived(t, svc, app, cipher)
 }
 
-func doTestSendNotificationPaymentSent(t *testing.T, svc *tests.TestService, app *db.App, ss []byte) {
+func TestSendNotification_LegacyApp_PaymentReceived(t *testing.T) {
```

Comment:
It would be good to change `LegacyApp` to `NoWalletPubkey` and do tests for both NIP04 and NIP44

Reply:
> NoWalletPubkey

I got confused reading this comment thinking "how is it related to `The public key does not have a wallet connected` test", can we have a different name?

Reply:
Does this sound good to you:
- `WithNip04Encryption`
- `WithNip44Encryption`
- `SharedWalletPubkey_WithNip04Encryption`
- `SharedWalletPubkey_WithNip44Encryption`